src/gtk/widgets/recent_info.rs

```diff
@@ -141,23 +142,22 @@ impl RecentInfo {
 
     pub fn has_group(&self, group_name: &str) -> bool {
         unsafe {
-            to_bool(
-                ffi::gtk_recent_info_has_group(GTK_RECENT_INFO(self.unwrap_widget()),
-                                               group_name.borrow_to_glib().0))
+            to_bool(ffi::gtk_recent_info_has_group(GTK_RECENT_INFO(self.unwrap_widget()),
+                group_name.borrow_to_glib().0))
         }
```

Comment:
Sure, but that still does not change the fact that the code is not conform to other aspects of code style, in this case, the hundred column rule.